packages/core/core/src/factories/useCartFactory.ts

```diff
@@ -34,6 +34,7 @@ export const useCartFactory = <CART, CART_ITEM, PRODUCT, COUPON>(
     const loading: Ref<boolean> = sharedRef(false, 'useCart-loading');
     const cart: Ref<CART> = sharedRef(null, 'useCart-cart');
     const context = generateContext(factoryParams);
+    const error: Ref<ComposableErrors> = sharedRef({}, 'useCart-error');
```

Comment:
IMHO this should be strictly typed per composable to allow autocompletion. I find this being one of the biggest values of this API, wdyt?

Reply:
That's a good idea